frontend-react/.storybook/preview.js

```diff
@@ -0,0 +1,59 @@
+import { initializeWorker, mswDecorator } from "msw-storybook-addon";
+import { CacheProvider } from "rest-hooks";
+import { BrowserRouter } from "react-router-dom";
+import { HelmetProvider } from "react-helmet-async";
+import MockDate from "mockdate";
+
+import "../src/content/generated/global.out.css";
+
+// mock all dates in stories to make sure we don't run into date-related inconsistencies
+MockDate.set("2023-01-01");
+
+export const parameters = {
+    actions: { argTypesRegex: "^on[A-Z].*" },
+    controls: {
+        matchers: {
+            color: /(background|color)$/i,
+            date: /Date$/,
+        },
+    },
+    options: {
+        storySort: {
+            method: "alphabetical",
+            order: [],
+        },
+    },
+};
+
+initializeWorker();
+
+function withRestHooksCacheProvider(Story) {
```

Comment:
We could add an all-in-one HOC like we do in tests, but I figure the granularity might be good.  Not really an issue either way, though!